app/assets/stylesheets/snowcrash/modules/index_table.scss

```diff
@@ -0,0 +1,53 @@
+.btn-toolbar {
+  input[type='checkbox'] {
+    margin: 0;
+  }
+
+  .js-index-table-actions {
```

Comment:
let's use a non-js class for css

Reply:
since we are already forcing ourselves to add that class in the html to make the table js work, I think that we can use the same class to make the table css work, instead of adding another `index-table-actions` class or similar 
Thoughts?

Reply:
sounds like a good idea 👍

Reply:
@dormi looking at this again, i feel like it would be better to use a class not prepended by `js-`. We should use `js-` classes for strictly js behavior only. What do you think of this convention?